test/lib/index.test.js

```diff
@@ -44,3 +44,38 @@ describe('ImageAsset', function() {
         expect(data).to.equal('asdf');
     });
 });
+
+describe('NewsAsset', function() {
+    it('can serialize out correctly', function() {
+        const asset = new libingester.NewsArticle();
+        asset.set_title('Test Asset');
+        asset.set_license('Proprietary');
+        asset.set_canonical_uri('https://www.example.com/');
+        asset.set_last_modified_date(new Date(1492545280000));
+        asset.set_document('<h1>Word of the Day</h1>');
+        asset.set_section("word_of_day");
+        asset.set_synopsis('a long time ago...');
+
+        const metadata = asset.to_metadata();
+
+        // Remove randomness -- should probably be a mock if I can
+        // figure out how to use it.
+        delete metadata['assetID'];
```

Comment:
Thumbnail should be handled here too, otherwise the test will not pass. @sgnn7, @wgreenberg: would you prefer the field not be output at all if no thumbnail is set - like in the change above for synopsis?